docs/xaml-tools/xaml-hot-reload.md

```diff
@@ -23,7 +23,7 @@ With XAML Hot Reload, you can incrementally build and test XAML code for your WP
 
 ## Where to get XAML Hot Reload
 
-Visual Studio XAML Hot Reload is currently supported only when you're running an application in **Visual Studio** -or- **Blend for Visual Studio** with the debugger attached (**F5** or **Start debugging**).
+Visual Studio XAML Hot Reload is currently supported only when you're running an application in **Visual Studio** -or- **Blend for Visual Studio** (If supported by the framework) with the debugger attached (**F5** or **Start debugging**).
```

Comment:
"If supported" => "if supported" (as it's part of the same sentence). But what cases aren't supported - what was the driver for adding this caveat?

Reply:
the Blend for Visual Studio..... is kinda odd... i wasn't sure how to word it... maybe remove Blend? basically i don't think blend supports some of the frameworks.... i assume?